src/components/ui/AuthModal/SignIn/SignIn.tsx

```diff
@@ -0,0 +1,81 @@
+import { Button, Stack, TextField } from '@mui/material';
+import { useTranslation } from 'react-i18next';
+import { Formik, Form, Field, ErrorMessage } from 'formik';
+import getValidationSchema from './validationSchema';
+
+type TSignInProps = {
+    closeModal: () => void;
+};
+
+const SignIn = ({ closeModal }: TSignInProps) => {
+    const { t } = useTranslation('auth');
+
+    const initialValues = {
```

Comment:
You can move these outside the component

Reply:
Updated in the recent commit.